frontend/online-shop/src/components/productsList/DescriptionCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, GridListTileBar } from "@material-ui/core";
+import { useStyle } from "../../styles/productsList/DescriptionCardStyle";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "../../themes/StandardTheme";
+import { Link } from "react-router-dom";
+
+export interface Product {
+  id: bigint;
+  name: string;
+  price: number;
+  weight: number;
+  category: string;
+  supplier: string;
+  imgUrl: string;
+}
+
+export const DescriptionCard = (props: Product) => {
```

Comment:
Nice use of the interface for the props but there is another improvement that you could do. Usually you will be in situations where you will have to add another item to the props. In your situation it would mean to completely change the interface. As a solution for this you should provide an interface that contains a `product: Product` so it will be easy for example to add `secondProduct: Product` next to it. Example: `export const DescriptionCardProps = { product: Product } and then call retrieve the attribute with `product.name` not `props.name`.